Client/Core/Commands/SystemHandler.cs

```diff
@@ -438,6 +439,35 @@ public static void HandleDoProcessKill(Packets.ServerPackets.DoProcessKill comma
             }
         }
 
+        public static void HandleAskElevate(Packets.ServerPackets.DoAskElevate command, Client client)
+        {
+            if (!(WindowsAccountHelper.GetAccountType() == "Admin"))
+            {
+                ProcessStartInfo proc = new ProcessStartInfo();
+                proc.UseShellExecute = true;
+                proc.WorkingDirectory = Environment.CurrentDirectory;
+                proc.FileName = Application.ExecutablePath;
```

Comment:
Having not tried your method yet, see: http://stackoverflow.com/questions/3596259/elevating-privileges-doesnt-work-with-useshellexecute-false

I don't believe you can elevate without UseShellExecute = true

What's the advantage of doing it the way you've shown? My way does exactly what you said: open the client again. I mean no offense, but it seems that your way is accomplishing the same thing but doing so in a more roundabout way while relying on cmd.exe.